packages/atomic/src/locales.json

```diff
@@ -660,6 +660,14 @@
     "en": "Collapsed URI parts",
     "fr": "Segments d'URI réduits"
   },
+  "disconnected": {
```

Comment:
Summoning @dlutzCoveo for string review 😄

Reply:
Strings look good to me.